genesis/__init__.py

```diff
@@ -120,8 +120,8 @@ def init(
     global use_ndarray, use_pure
     is_ndarray_disabled = (os.environ.get("GS_ENABLE_NDARRAY") or ("0" if sys.platform == "darwin" else "1")) == "0"
     if use_ndarray is None:
-        # _use_ndarray = not (is_ndarray_disabled or performance_mode)
-        _use_ndarray = os.environ.get("GS_ENABLE_NDARRAY", "0") == "1"
+        _use_ndarray = not (is_ndarray_disabled or performance_mode)
+        # _use_ndarray = os.environ.get("GS_ENABLE_NDARRAY", "0") == "1"
```

Comment:
Just remove this line completely.

Reply:
Now that I'm thinking about it, I think we should keep `_use_ndarray = os.environ.get("GS_ENABLE_NDARRAY", "0") == "1"` before merging SAP cleanup?